patches/tModLoader/Terraria/release_extras/LaunchUtils/ScriptCaller.sh

```diff
@@ -66,6 +72,10 @@ else
 		echo "A Windows dotnet executable was detected, possibly from a previous Proton launch. Deleting dotnet_dir and resetting"  2>&1 | tee -a "$LogFile"
 		rm -rf "$dotnet_dir"
 		mkdir "$dotnet_dir"
+	elif [[ "$(file "$install_dir/dotnet")" == *"arm64"* ]]; then
+		echo "An arm64 install of dotnet was detected. Deleting dotnet_dir and resetting"  2>&1 | tee -a "$LogFile"
```

Comment:
Need to limit to Darwin Arm64.
Linux Arm64 is still valid and would loop here

Reply:
Better to put a `"$_arch" != "arm64"`

Reply:
Further, if '$(file ....) != $_arch' if we are being fastidious to that level?

Reply:
There's not a guarantee atm that `file` will output something in the exact same form as `uname -m` so I'm only handling the "`arm64` was installed by accident" case